app/views/home/index.html.haml

```diff
@@ -6,7 +6,10 @@
   %nav#top_nav.my-2.my-md-0.mr-md-3
     %a.p-2.text-dark{:href => "#"} About Us
     %a.p-2.text-dark{:href => "#"} Contact
-    %a.p-2.text-dark{:href => dashboard_root_path} Dashboard
+    - if user_signed_in?
```

Comment:
Here we can just use ‘link_to’ right? Specify the class as ‘p-2’ and ‘text-dark’ and put the conditional as the second argument